### ISSUE TITLE
Faster image loading time

### ISSUE THREAD

**Issue by GarbatyGrabarz** [pierosavi/pierosavi-imageit-panel#82]
**Problem**: It takes a lot of time to load an image in the panel. In my case it takes over 8 seconds. Since the panel is on a front page it looks pretty bad when the whole page with logs and shortcuts loads but the crown jewel of the image still has not loaded.

**What should happen**: Image should load as fast as other panels.

**Tried alternatives**: I have created a panel with the same image but no sensor data and it loaded the image instantly.

**Notes**: It would improve user experience if the image was loaded before reading/accessing/processing sensor information.

**Comment by GarbatyGrabarz** [comment]
@pierosavi I use InfluxDB

**Comment by pierosavi** [comment]
I tried improving times by loading only the image at first but it doesn't seem possible as it's a Grafana limitation
To debug I tried turning off my influxdb 2 instance and add the same 2 queries to 2 different panels, one imageit and one default Grafana panel like Stats, then reload the page
They both stay completely blank for 5-6 seconds before showing "no data"
I see that the plugin code doesn't get executed at all during that time
Can you try with your data source the same test and see if you have the same result?

**Comment by GarbatyGrabarz** [comment]
Debugging "resolved" the issue. It was the human error. I was shamelessly comparing loading time of a small database (events log) with sensor data collected every 30 seconds for a period of last 90 days. I have added a panel as you suggested and that had lead me to the discovery of my stupid error.

I have resolved my issue by setting a short time range since there is no graph involved and used custom time range for the events panel. Thank you, @pierosavi for your help! :)
